homeassistant/components/godice/sensor.py

```diff
@@ -0,0 +1,130 @@
+"""Sensors for GoDice device."""
+
+import logging
+
+import godice
+
+from homeassistant import const
+from homeassistant.components.sensor import (
+    RestoreSensor,
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import DiceConfigEntry
+from .const import DOMAIN, MANUFACTURER, MODEL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+ROLLED_NUMBER_SENSOR_DESCR = SensorEntityDescription(
+    key="rolled_number", state_class=SensorStateClass.MEASUREMENT
+)
```

Comment:
Would this be a nice enum sensor as well? Since you probably know all the possible values

Reply:
changed to enum, please check